src/game/level.rs

```diff
@@ -0,0 +1,31 @@
+//! Setup the level
+
+use bevy::prelude::*;
+
+use super::physics::{PhysicalTransform, Velocity};
+use crate::screen::Screen;
+
+#[derive(Debug, Event)]
+pub(crate) struct SpawnLevel;
+
+pub(super) fn plugin(app: &mut App) {
+    app.observe(spawn_level);
+}
+
+fn spawn_level(
```

Comment:
Maybe level -> player?
There is no level really

Reply:
I'd leave it as-is so that users know where to put their level code.

Reply:
Discussed on discord https://discord.com/channels/691052431525675048/1258521739395203174/1259897153506115725

Reply:
Per discussion on Discord, I changed it: [`cd4934f` (#70)](https://github.com/TheBevyFlock/bevy-template/pull/70/commits/cd4934f0536b731875d1150e1bc83b9691130b8a)